src/c4v/classifier/classifier.py

```diff
@@ -0,0 +1,381 @@
+"""
+    This classifier class will perform basic experiments, receiving 
+    as arguments the training arguments and the columns to use from the training dataset
+"""
+# Python imports
+from typing             import Dict, List, Any, Tuple
+from pathlib            import Path
+from importlib          import resources
+from enum               import Enum
+# Third Party
+from pandas.core.frame  import DataFrame
+from transformers import (
+    RobertaTokenizer,
+    Trainer,
+    TrainingArguments,
+    RobertaForSequenceClassification,
+)
+from sklearn.metrics import accuracy_score, f1_score, recall_score, precision_score
+from sklearn.model_selection import train_test_split
+from datasets import Dataset
+import torch
+import pandas as pd
+import numpy as np
+import os
+
+from transformers.trainer_utils import EvalPrediction
+
+device = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
+
+BASE_C4V_FOLDER = os.path.join(os.environ.get("HOME"), "/.c4v")
```

Comment:
What about having all paths as settings?

Reply:
Yes, I have to change it, this was done before having dynaconf settings :)